editor/editor_help.cpp

```diff
@@ -172,7 +172,9 @@ void EditorHelp::_class_desc_input(const Ref<InputEvent> &p_input) {
 void EditorHelp::_class_desc_resized() {
 	// Add extra horizontal margins for better readability.
 	// The margins increase as the width of the editor help container increases.
-	const int display_margin = MAX(30 * EDSCALE, get_parent_anchorable_rect().size.width - 900 * EDSCALE) * 0.5;
+	Ref<Font> doc_code_font = get_font("doc_source", "EditorFonts");
+	real_t char_width = doc_code_font->get_char_size(' ').width;
```

Comment:
Wouldn't the width of the `x` character be more appropriate? This may be important if the user uses a proportional source font.

Reply:
good point. i agree.

Reply:
updated. :smile: